gems/pending/Gemfile

```diff
@@ -46,9 +46,7 @@ gem "sys-uname",               "~>1.0.1",           :require => false
 gem "trollop",                 "~>2.0",             :require => false
 gem "uuidtools",               "~>2.1.3",           :require => false
 
-if RbConfig::CONFIG["host_os"].include?("darwin") # This is only used on Mac
-  gem 'sys-proctable',  "~> 0.9",      :require => false
-end
+gem 'sys-proctable',  :github => "djberg96/sys-proctable", :branch => :master
```

Comment:
@djberg96 I tested this and it looks good!  We can release a new gem and I'll change this to use that version.  Thanks!